.circleci/config.yml

```diff
@@ -49,7 +49,7 @@ jobs:
             mkdir -p $HOME/artifacts/junit/
             find . -type f -regex ".*/spring-cloud-*.*/target/*.*" -exec cp {} $HOME/artifacts/ \;
             find . -type f -regex ".*/target/.*-reports/.*" -exec cp {} $HOME/artifacts/junit/ \;
-            bash <(curl -s https://codecov.io/bash)
+            # bash <(curl -s https://codecov.io/bash)
```

Comment:
https://github.com/spring-cloud/spring-cloud-kubernetes/pull/946/files#r774928963

Reply:
Lets worry about this in another PR as it has nothing to do with resolving this issue

Reply:
you are completely right. I disabled it, because otherwise the build would fail. I guess it happens because the percentage decreases somehow because of the changes I made? What I mean by this: I can re-enable it, of course, but then the build will fail.

Reply:
Thats fine, remove the change and we will address it in another PR

Reply:
done, but as said the build now fails

Reply:
But the build passed ;)

Reply:
oh! I see your point. I can be really dumb at times, sorry about that :)

Reply:
No way not at all, I am being a smart ass!  Thanks for everything you do and contribute!